Detectors/TRD/base/src/TRDGeometry.cxx

```diff
@@ -2645,7 +2645,7 @@ void TRDGeometry::addAlignableVolumes() const
   }
 
   std::string volPath;
-  std::string vpStr{"ALIC_1/B077_1/BSEGMO"};
+  std::string vpStr{"ALIC_1/barrel_1/B077_1/BSEGMO"};
```

Comment:
Hi @amorsch, 
This change actually broke functionality at the TRD. I was having 1/18 of all TRD chambers and have no idea why.  The following method:
`GPUd() static int getDetector(int layer, int stack, int sector) { return (layer + stack * kNlayer + sector * kNlayer * kNstack); }` in https://github.com/AliceO2Group/AliceO2/blob/9372db278daa40510510371add926c7879ec5e78/Detectors/TRD/base/include/TRDBase/TRDGeometryBase.h#L44 changed its behavior. I am trying to find out how this affects others part of the code to revert the problem.

Reply:
Fixed in https://github.com/AliceO2Group/AliceO2/pull/3890.